pkg/util/files.go

```diff
@@ -222,6 +226,43 @@ func fetchAutoDeps(repo string, dep *model.Dependency, dest string) error {
 	return nil
 }
 
+func fetchAutoProxyWorkspace(dep *model.Dependency, dest string) error {
+	wsFile, err := os.Open(path.Join(dest, "../proxy/WORKSPACE"))
```

Comment:
I think the code below could be simplified by using `ioutil.ReadFile` and [regular expressions](https://golang.org/pkg/regexp/#Regexp.FindString).

Reply:
Thanks, done.